### ISSUE TITLE
After importing new settings (presets etc.) the UI does not update until restart

### ISSUE THREAD

**Issue by aliya-ryan** [digidem/mapeo-desktop#33]
Currently if you import new settings "importar configurarcion" you then have to close mapeo and reopen it in order for them to be visible. 

Could there be a refresh button or automatic refresh?

**Comment by noffle** [comment]
@aliya-ryan: Is this still true? What settings aren't you seeing refreshed?

I tried the following:

1. `rm imagery.json icons.svg presets.json translations.json`
2. run mapeo
3. create a new node, see the stock presents (hospital, park, etc)
4. import waorani settings
5. click on node again, see all of the waorani presets

**Comment by noffle** [comment]
I poked at this again. It looks like the icons refresh only if you're going from "no icon set" to some icon, for presets. Since I went from defaults -> waorani I saw the icons change and it all looked good to me; however, doing e.g. seikopai -> waoroni wasn't working.

This seems to be an issue with iD editor. As a workaround, I made the window refresh itself after loading in new settings, which is relatively seemless compared to a full restart: https://github.com/digidem/mapeo-desktop/pull/92
